### ISSUE TITLE
[Help] Buzzer doesn't work

### ISSUE THREAD

**Issue by JaroslawPrzybyl** [joBr99/nspanel-lovelace-ui#505]
### PROBLEM DESCRIPTION

I'm trying to use notifications with sound , but buzzer on both my devices doesn't work as expected. By command buzzer 1,1,1 I should hear one 100ms beep, right? I hear only silent click-click, like begin and end of expected sound, but no tone in between. PWM is not working? Where I can check it?

### REQUESTED INFORMATION

- [ ] Provide a screenshot of your tasmota mqtt configuration:
![image](https://user-images.githubusercontent.com/33628620/194358708-4c3b2f08-bdb6-4186-9723-bb476c718c34.png)

- [ ] Provide your appdaemon.yaml: 
```
  appdaemon.yaml here:
`---
secrets: /config/secrets.yaml
appdaemon:
  latitude: 52.37xxx
  longitude: 4.89xxx
  elevation: 2
  time_zone: Europe/Amsterdam
  plugins:
    HASS:
      type: hass
    MQTT:
      type: mqtt
      namespace: mqtt
      client_id: "appdaemon"
      client_host: 10.4.4.10
      client_port: 1883
      client_user: "mqttuser"
      client_password: [pwd]
      client_topics: NONE
http:
  url: http://127.0.0.1:5050
admin:
api:
hadashboard:
`

```

- [ ] Provide your apps.yaml:
```
  apps.yaml here:
`nspanel-2:
  module: nspanel-lovelace-ui
  class: NsPanelLovelaceUIManager
  config:
    panelRecvTopic: "tele/NSPanel_Kuchnia/RESULT"
    panelSendTopic: "cmnd/NSPanel_Kuchnia/CustomSend"
    sleepTimeout: 20
    sleepBrightness:
      - time: "7:00:00"
        value: 10
      - time: "23:00:00"
        value: 3
    locale: "pl_PL"
    screensaver:
      entity: weather.dom_w_lesie
      doubleTapToUnlock: true
      alternativeLayout: true 
      theme:
          autoWeather: true
          time:             [102, 255, 51]
        #timeAMPM:         [220, 0, 255]
          date:             [255, 255, 0]
    cards:
      - type: cardGrid
        title: Przed domem
        entities:
          - entity: switch.oswietlenie_zewnetrzne_1
            name: Ścieżka
            icon: mdi:outdoor-lamp
          - entity: switch.oswietlenie_zewnetrzne_2
            name: LED Zewn.
            icon: mdi:theme-light-dark
          - entity: switch.oswietlenie_zewnetrzne_3
            name: Podjazd
            icon: mdi:coach-lamp          `

```
- [ ] Go to Settings>Add-ons>AppDaemon>Log and then, provide the output of the log after restarting the appdaemon container:
```
  Log output here:
`[17:49:56] INFO: Starting AppDaemon...
2022-10-06 17:50:07.838405 INFO AppDaemon: AppDaemon Version 4.2.1 starting
2022-10-06 17:50:07.839299 INFO AppDaemon: Python version is 3.10.5
2022-10-06 17:50:07.840099 INFO AppDaemon: Configuration read from: /config/appdaemon/appdaemon.yaml
2022-10-06 17:50:07.840979 INFO AppDaemon: Added log: AppDaemon
2022-10-06 17:50:07.842205 INFO AppDaemon: Added log: Error
2022-10-06 17:50:07.842969 INFO AppDaemon: Added log: Access
2022-10-06 17:50:07.843582 INFO AppDaemon: Added log: Diag
2022-10-06 17:50:08.241609 INFO AppDaemon: Loading Plugin HASS using class HassPlugin from module hassplugin
2022-10-06 17:50:08.711825 INFO HASS: HASS Plugin Initializing
2022-10-06 17:50:08.712332 INFO HASS: HASS Plugin initialization complete
2022-10-06 17:50:08.713315 INFO AppDaemon: Loading Plugin MQTT using class MqttPlugin from module mqttplugin
2022-10-06 17:50:08.991465 INFO MQTT: MQTT Plugin Initializing
2022-10-06 17:50:08.992080 INFO MQTT: Using 'appdaemon/status' as Will Topic
2022-10-06 17:50:08.992551 INFO MQTT: Using 'appdaemon/status' as Birth Topic
2022-10-06 17:50:08.994007 INFO AppDaemon: Initializing HTTP
2022-10-06 17:50:08.994977 INFO AppDaemon: Using 'ws' for event stream
2022-10-06 17:50:09.008064 INFO AppDaemon: Starting API
2022-10-06 17:50:09.016814 INFO AppDaemon: Starting Admin Interface
2022-10-06 17:50:09.017879 INFO AppDaemon: Starting Dashboards
2022-10-06 17:50:09.062556 INFO HASS: Connected to Home Assistant 2022.9.7
2022-10-06 17:50:09.147720 INFO MQTT: Connected to Broker at URL 10.4.4.10:1883
2022-10-06 17:50:09.233069 INFO AppDaemon: Got initial state from namespace mqtt
2022-10-06 17:50:09.234399 INFO MQTT: MQTT Plugin initialization complete
2022-10-06 17:50:09.321923 INFO HASS: Evaluating startup conditions
2022-10-06 17:50:09.331294 INFO AppDaemon: App 'nspanel-1' added
2022-10-06 17:50:09.335723 INFO AppDaemon: App 'nspanel-2' added
2022-10-06 17:50:09.340833 INFO AppDaemon: Found 2 total apps
2022-10-06 17:50:09.342339 INFO AppDaemon: Starting Apps with 2 workers and 2 pins
2022-10-06 17:50:09.347875 INFO AppDaemon: Running on port 5050
2022-10-06 17:50:09.362034 INFO HASS: Startup condition met: hass state=RUNNING
2022-10-06 17:50:09.363643 INFO HASS: All startup conditions met
2022-10-06 17:50:09.464119 INFO AppDaemon: Got initial state from namespace default
2022-10-06 17:50:11.368423 INFO AppDaemon: Scheduler running in realtime
2022-10-06 17:50:11.377982 INFO AppDaemon: Adding /config/appdaemon/apps to module import path
2022-10-06 17:50:11.379708 INFO AppDaemon: Adding /config/appdaemon/apps/nspanel-lovelace-ui to module import path
2022-10-06 17:50:11.381413 INFO AppDaemon: Adding /config/appdaemon/apps/nspanel-lovelace-ui/luibackend to module import path
2022-10-06 17:50:11.382840 INFO AppDaemon: Adding /config/appdaemon/apps/nspanel-lovelace-ui/luibackend/translations to module import path
2022-10-06 17:50:11.384696 INFO AppDaemon: Adding /config/appdaemon/apps/nspanel-lovelace-ui/luibackend/translations/frontend to module import path
2022-10-06 17:50:11.386293 INFO AppDaemon: Adding /config/appdaemon/apps/nspanel-lovelace-ui/luibackend/translations/backend to module import path
2022-10-06 17:50:11.413875 WARNING AppDaemon: No app description found for: /config/appdaemon/apps/hello.py - ignoring
2022-10-06 17:50:11.416944 INFO AppDaemon: Loading App Module: /config/appdaemon/apps/nspanel-lovelace-ui/nspanel-lovelace-ui.py
2022-10-06 17:50:11.928495 WARNING AppDaemon: No app description found for: /config/appdaemon/apps/nspanel-lovelace-ui/luibackend/updater.py - ignoring
2022-10-06 17:50:11.930921 WARNING AppDaemon: No app description found for: /config/appdaemon/apps/nspanel-lovelace-ui/luibackend/icon_mapping.py - ignoring
2022-10-06 17:50:11.933444 WARNING AppDaemon: No app description found for: /config/appdaemon/apps/nspanel-lovelace-ui/luibackend/config.py - ignoring
2022-10-06 17:50:11.935756 WARNING AppDaemon: No app description found for: /config/appdaemon/apps/nspanel-lovelace-ui/luibackend/controller.py - ignoring
2022-10-06 17:50:11.938527 WARNING AppDaemon: No app description found for: /config/appdaemon/apps/nspanel-lovelace-ui/luibackend/localization.py - ignoring
2022-10-06 17:50:11.940910 WARNING AppDaemon: No app description found for: /config/appdaemon/apps/nspanel-lovelace-ui/luibackend/pages.py - ignoring
2022-10-06 17:50:11.943535 WARNING AppDaemon: No app description found for: /config/appdaemon/apps/nspanel-lovelace-ui/luibackend/helper.py - ignoring
2022-10-06 17:50:11.946021 WARNING AppDaemon: No app description found for: /config/appdaemon/apps/nspanel-lovelace-ui/luibackend/mqtt.py - ignoring
2022-10-06 17:50:11.948645 WARNING AppDaemon: No app description found for: /config/appdaemon/apps/nspanel-lovelace-ui/luibackend/theme.py - ignoring
2022-10-06 17:50:11.952933 WARNING AppDaemon: No app description found for: /config/appdaemon/apps/nspanel-lovelace-ui/luibackend/apis.py - ignoring
2022-10-06 17:50:11.955634 WARNING AppDaemon: No app description found for: /config/appdaemon/apps/nspanel-lovelace-ui/luibackend/icons.py - ignoring
2022-10-06 17:50:11.958220 INFO AppDaemon: Initializing app nspanel-1 using class NsPanelLovelaceUIManager from module nspanel-lovelace-ui
2022-10-06 17:50:11.964960 INFO AppDaemon: Initializing app nspanel-2 using class NsPanelLovelaceUIManager from module nspanel-lovelace-ui
2022-10-06 17:50:12.183336 INFO nspanel-1: Starting
2022-10-06 17:50:12.199326 INFO nspanel-1: Input config: {'panelRecvTopic': 'tele/NSPanel_warsztat/RESULT', 'panelSendTopic': 'cmnd/NSPanel_warsztat/CustomSend', 'sleepTimeout': 20, 'sleepBrightness': [{'time': '7:00:00', 'value': 10}, {'time': '23:00:00', 'value': 3}], 'locale': 'pl_PL', 'screensaver': {'entity': 'weather.dom_w_lesie', 'doubleTapToUnlock': True, 'alternativeLayout': True, 'theme': {'time': [102, 255, 51], 'date': [255, 255, 0], 'bar': [0, 0, 255], 'clear-night': [150, 150, 100], 'cloudy': [255, 204, 255], 'exceptional': [255, 50, 50], 'fog': [150, 150, 150], 'hail': [200, 200, 200], 'lightning': [200, 200, 0], 'lightning-rainy': [200, 200, 150], 'partlycloudy': [150, 150, 150], 'pouring': [50, 50, 255], 'rainy': [100, 100, 255], 'snowy': [150, 150, 150], 'snowy-rainy': [150, 150, 255], 'sunny': [255, 255, 0], 'windy': [150, 150, 150], 'windy-variant': [255, 125, 125]}}, 'cards': [{'type': 'cardGrid', 'title': 'Osoby', 'entities': [{'entity': 'person.jarek', 'icon': 'mdi:face-man'}, {'entity': 'person.asia', 'icon': 'mdi:face-woman'}, {'entity': 'person.kuba', 'icon': 'mdi:face-man-profile'}, {'entity': 'person.franek', 'icon': 'mdi:face-woman-profile'}]}]}
2022-10-06 17:50:12.204429 INFO nspanel-1: Loaded config: {'panelRecvTopic': 'tele/NSPanel_warsztat/RESULT', 'panelSendTopic': 'cmnd/NSPanel_warsztat/CustomSend', 'updateMode': 'auto-notify', 'model': 'eu', 'sleepTimeout': 20, 'sleepBrightness': [{'time': '7:00:00', 'value': 10}, {'time': '23:00:00', 'value': 3}], 'screenBrightness': 100, 'defaultBackgroundColor': 'ha-dark', 'sleepTracking': None, 'sleepTrackingZones': ['not_home', 'off'], 'sleepOverride': None, 'locale': 'pl_PL', 'timeFormat': '%H:%M', 'dateFormatBabel': 'full', 'dateAdditionalTemplate': '', 'timeAdditionalTemplate': '', 'dateFormat': '%A, %d. %B %Y', 'cards': [{'type': 'cardGrid', 'title': 'Osoby', 'entities': [{'entity': 'person.jarek', 'icon': 'mdi:face-man'}, {'entity': 'person.asia', 'icon': 'mdi:face-woman'}, {'entity': 'person.kuba', 'icon': 'mdi:face-man-profile'}, {'entity': 'person.franek', 'icon': 'mdi:face-woman-profile'}]}], 'screensaver': {'type': 'screensaver', 'entity': 'weather.dom_w_lesie', 'weatherUnit': 'celsius', 'forecastSkip': 0, 'weatherOverrideForecast1': None, 'weatherOverrideForecast2': None, 'weatherOverrideForecast3': None, 'weatherOverrideForecast4': None, 'doubleTapToUnlock': True, 'alternativeLayout': True, 'defaultCard': None, 'key': 'screensaver', 'theme': {'time': [102, 255, 51], 'date': [255, 255, 0], 'bar': [0, 0, 255], 'clear-night': [150, 150, 100], 'cloudy': [255, 204, 255], 'exceptional': [255, 50, 50], 'fog': [150, 150, 150], 'hail': [200, 200, 200], 'lightning': [200, 200, 0], 'lightning-rainy': [200, 200, 150], 'partlycloudy': [150, 150, 150], 'pouring': [50, 50, 255], 'rainy': [100, 100, 255], 'snowy': [150, 150, 150], 'snowy-rainy': [150, 150, 255], 'sunny': [255, 255, 0], 'windy': [150, 150, 150], 'windy-variant': [255, 125, 125]}}, 'hiddenCards': []}
2022-10-06 17:50:12.216871 INFO nspanel-1: Sending MQTT Message: pageType~pageStartup
2022-10-06 17:50:12.269354 INFO nspanel-1: Registering callbacks for the following items: ['person.jarek', 'person.asia', 'person.kuba', 'person.franek', 'weather.dom_w_lesie']
2022-10-06 17:50:12.305368 INFO nspanel-1: Started
2022-10-06 17:50:12.318633 INFO nspanel-2: Starting
2022-10-06 17:50:12.329426 INFO nspanel-2: Input config: {'panelRecvTopic': 'tele/NSPanel_Kuchnia/RESULT', 'panelSendTopic': 'cmnd/NSPanel_Kuchnia/CustomSend', 'sleepTimeout': 20, 'sleepBrightness': [{'time': '7:00:00', 'value': 10}, {'time': '23:00:00', 'value': 3}], 'locale': 'pl_PL', 'screensaver': {'entity': 'weather.dom_w_lesie', 'doubleTapToUnlock': True, 'alternativeLayout': True, 'theme': {'autoWeather': True, 'time': [102, 255, 51], 'date': [255, 255, 0], 'bar': [0, 0, 255], 'clear-night': [150, 150, 100], 'cloudy': [255, 204, 255], 'exceptional': [255, 50, 50], 'fog': [150, 150, 150], 'hail': [200, 200, 200], 'lightning': [200, 200, 0], 'lightning-rainy': [200, 200, 150], 'partlycloudy': [150, 150, 150], 'pouring': [50, 50, 255], 'rainy': [100, 100, 255], 'snowy': [150, 150, 150], 'snowy-rainy': [150, 150, 255], 'sunny': [255, 255, 0], 'windy': [150, 150, 150], 'windy-variant': [255, 125, 125]}}, 'cards': [{'type': 'cardGrid', 'title': 'Przed domem', 'entities': [{'entity': 'switch.oswietlenie_zewnetrzne_1', 'name': 'Ścieżka', 'icon': 'mdi:outdoor-lamp'}, {'entity': 'switch.oswietlenie_zewnetrzne_2', 'name': 'LED Zewn.', 'icon': 'mdi:theme-light-dark'}, {'entity': 'switch.oswietlenie_zewnetrzne_3', 'name': 'Podjazd', 'icon': 'mdi:coach-lamp'}]}]}
2022-10-06 17:50:12.334328 INFO nspanel-2: Loaded config: {'panelRecvTopic': 'tele/NSPanel_Kuchnia/RESULT', 'panelSendTopic': 'cmnd/NSPanel_Kuchnia/CustomSend', 'updateMode': 'auto-notify', 'model': 'eu', 'sleepTimeout': 20, 'sleepBrightness': [{'time': '7:00:00', 'value': 10}, {'time': '23:00:00', 'value': 3}], 'screenBrightness': 100, 'defaultBackgroundColor': 'ha-dark', 'sleepTracking': None, 'sleepTrackingZones': ['not_home', 'off'], 'sleepOverride': None, 'locale': 'pl_PL', 'timeFormat': '%H:%M', 'dateFormatBabel': 'full', 'dateAdditionalTemplate': '', 'timeAdditionalTemplate': '', 'dateFormat': '%A, %d. %B %Y', 'cards': [{'type': 'cardGrid', 'title': 'Przed domem', 'entities': [{'entity': 'switch.oswietlenie_zewnetrzne_1', 'name': 'Ścieżka', 'icon': 'mdi:outdoor-lamp'}, {'entity': 'switch.oswietlenie_zewnetrzne_2', 'name': 'LED Zewn.', 'icon': 'mdi:theme-light-dark'}, {'entity': 'switch.oswietlenie_zewnetrzne_3', 'name': 'Podjazd', 'icon': 'mdi:coach-lamp'}]}], 'screensaver': {'type': 'screensaver', 'entity': 'weather.dom_w_lesie', 'weatherUnit': 'celsius', 'forecastSkip': 0, 'weatherOverrideForecast1': None, 'weatherOverrideForecast2': None, 'weatherOverrideForecast3': None, 'weatherOverrideForecast4': None, 'doubleTapToUnlock': True, 'alternativeLayout': True, 'defaultCard': None, 'key': 'screensaver', 'theme': {'autoWeather': True, 'time': [102, 255, 51], 'date': [255, 255, 0], 'bar': [0, 0, 255], 'clear-night': [150, 150, 100], 'cloudy': [255, 204, 255], 'exceptional': [255, 50, 50], 'fog': [150, 150, 150], 'hail': [200, 200, 200], 'lightning': [200, 200, 0], 'lightning-rainy': [200, 200, 150], 'partlycloudy': [150, 150, 150], 'pouring': [50, 50, 255], 'rainy': [100, 100, 255], 'snowy': [150, 150, 150], 'snowy-rainy': [150, 150, 255], 'sunny': [255, 255, 0], 'windy': [150, 150, 150], 'windy-variant': [255, 125, 125]}}, 'hiddenCards': []}
2022-10-06 17:50:12.348392 INFO nspanel-2: Sending MQTT Message: pageType~pageStartup
2022-10-06 17:50:12.397941 INFO nspanel-1: MQTT callback for: {'topic': 'tele/NSPanel_warsztat/RESULT', 'wildcard': None, 'payload': '{"nlui_driver_version":"4"}'}
2022-10-06 17:50:12.400172 INFO nspanel-2: Registering callbacks for the following items: ['switch.oswietlenie_zewnetrzne_1', 'switch.oswietlenie_zewnetrzne_2', 'switch.oswietlenie_zewnetrzne_3', 'weather.dom_w_lesie']
2022-10-06 17:50:12.420007 INFO nspanel-1: Update Pre-Check failed Tasmota Driver Version: 4 Panel Version: None
2022-10-06 17:50:12.448764 INFO nspanel-2: Started
2022-10-06 17:50:12.452897 INFO AppDaemon: App initialization complete
2022-10-06 17:50:12.968520 INFO nspanel-2: MQTT callback for: {'topic': 'tele/NSPanel_Kuchnia/RESULT', 'wildcard': None, 'payload': '{"CustomRecv":"event,startup,42,eu"}'}
2022-10-06 17:50:12.994836 INFO nspanel-2: Received Message from Screen: event,startup,42,eu
2022-10-06 17:50:13.006026 INFO nspanel-1: MQTT callback for: {'topic': 'tele/NSPanel_warsztat/RESULT', 'wildcard': None, 'payload': '{"CustomRecv":"event,startup,42,eu"}'}
2022-10-06 17:50:13.022889 INFO nspanel-1: Received Message from Screen: event,startup,42,eu
2022-10-06 17:50:13.026450 INFO nspanel-2: Sending MQTT Message: X
2022-10-06 17:50:13.038903 INFO nspanel-1: Sending MQTT Message: X
2022-10-06 17:50:13.042427 INFO nspanel-2: Update Pre-Check failed Tasmota Driver Version: None Panel Version: 42
2022-10-06 17:50:13.054468 INFO nspanel-2: Startup Event
2022-10-06 17:50:13.056270 INFO nspanel-1: Update Pre-Check failed Tasmota Driver Version: None Panel Version: 42
2022-10-06 17:50:13.091527 INFO nspanel-2: Previous message repeated 1 times
2022-10-06 17:50:13.093320 INFO nspanel-2: Sending MQTT Message: time~17:50~
2022-10-06 17:50:13.144647 INFO nspanel-1: Sending MQTT Message: time~17:50~
2022-10-06 17:50:13.247192 INFO nspanel-2: Sending MQTT Message: date~czwartek, 6 pa��dziernika 2022
2022-10-06 17:50:13.258841 INFO nspanel-1: Sending MQTT Message: date~czwartek, 6 pa��dziernika 2022
2022-10-06 17:50:13.262025 INFO nspanel-2: Sending MQTT Message: timeout~20
2022-10-06 17:50:13.273428 INFO nspanel-1: Sending MQTT Message: timeout~20
2022-10-06 17:50:13.280185 INFO nspanel-2: Sending MQTT Message: dimmode~10~100~6371
2022-10-06 17:50:13.288494 INFO nspanel-1: Sending MQTT Message: dimmode~10~100~6371
2022-10-06 17:50:13.295049 INFO nspanel-2: Started rendering of page None with type screensaver
2022-10-06 17:50:13.304286 INFO nspanel-2: Previous message repeated 1 times
2022-10-06 17:50:13.305847 INFO nspanel-2: Sending MQTT Message: pageType~screensaver
2022-10-06 17:50:13.313665 INFO nspanel-1: Sending MQTT Message: pageType~screensaver
2022-10-06 17:50:13.391147 INFO nspanel-2: Sending MQTT Message: weatherUpdate~���~17.0��C~pt.~���~17.1��C~sob.~���~19.5��C~niedz.~���~14.2��C~pon.~���~14.5��C~���~67 %
2022-10-06 17:50:13.398717 INFO nspanel-1: Sending MQTT Message: weatherUpdate~���~17.0��C~pt.~���~17.1��C~sob.~���~19.5��C~niedz.~���~14.2��C~pon.~���~14.5��C~���~67 %
2022-10-06 17:50:13.411090 INFO nspanel-2: Sending MQTT Message: color~0~26598~65535~65504~65504~65535~65535~65535~65535~65535~65504~38066~38066~65504~65535~65535~65535~65535~31~65535~65535~65535
2022-10-06 17:50:13.417402 INFO nspanel-1: Sending MQTT Message: color~0~26598~65535~65504~65535~65535~65535~65535~65535~65535~65535~65535~65535~65535~65535~65535~65535~65535~31~65535~65535~65535
2022-10-06 17:50:13.473826 INFO nspanel-2: MQTT callback for: {'topic': 'tele/NSPanel_Kuchnia/RESULT', 'wildcard': None, 'payload': '{"nlui_driver_version":"4"}'}
2022-10-06 17:50:13.475415 INFO nspanel-1: MQTT callback for: {'topic': 'tele/NSPanel_warsztat/RESULT', 'wildcard': None, 'payload': '{"nlui_driver_version":"4"}'}
2022-10-06 17:50:13.480446 INFO nspanel-2: Update Pre-Check sucessful Tasmota Driver Version: 4 Panel Version: 42
2022-10-06 17:50:13.506994 INFO nspanel-1: Update Pre-Check sucessful Tasmota Driver Version: 4 Panel Version: 42`

```

### ADDITIONAL INFORMATION

_Add information about your setup here, if any. (For example docker version of AppDaemon instead of the HomeAssistant Add-on)_
Command, which I'm using for test:
`service: mqtt.publish
data:
  topic:  cmnd/NSPanel_Kuchnia/Backlog
  payload: CustomSend notify~Uwaga Uwaga~dupa jasio; Buzzer 1,1,1
`

**Comment by joBr99** [comment]
You can use `BuzzerPWM 1` to change the Operation Mode of the Buzzer, which increases the volume. (Not sure what Buzzer is actually in the NsPanel and if PWM Mode is the correct setting, but it increases the volume)

https://tasmota.github.io/docs/Commands/

https://tasmota.github.io/docs/Buzzer/

**Comment by JaroslawPrzybyl** [comment]
Thx! 
Command BuzzerPWM 1 enabled the PWM and now buzzer works a bit better. A bit because sound is really silent. I know, there is a small piezo beeper inside, but before flashing this device with tasmota I tested it with Esphome software and here notification sound was much louder. PWM has wrong settings for this piezo? Is there any possibility to change e.g. frequency of PWM? 

Also parameters for buzzer are working a bit different than manual says, difference is with fourth parameter. In linked article there is: "Parameters for the Buzzer command can be <count>,<beep>,<silence>,<tune>". I made several tests with different configurations of parameters and with one, two or three parameters it works as described. But fourth parameter - for sure it is not tune, sound is always the same, but it seems that this fourth parameter somehow modifies all the rest of parameters. Difficult to explain, just compare Buzzer 3,2,2,1 with Buzzer 3,2,2,5 and Buzzer 3,2,2,10
